src/SFA.DAS.LevyTransferMatching.Web/Orchestrators/IPledgeOrchestrator.cs

```diff
@@ -15,6 +15,6 @@ public interface IPledgeOrchestrator
         Task UpdateCacheItem(JobRolePostRequest request);
         Task UpdateCacheItem(LevelPostRequest request);
         Task<LevelViewModel> GetLevelViewModel(LevelRequest request);
-        Task SubmitPledge(CreatePostRequest request);
+        Task<ConfirmationViewModel> SubmitPledge(CreatePostRequest request);
```

Comment:
I don't think this method should return this type of object; as I said above I think it'd be cleaner to simply return the pledge id (encoded or otherwise).

Reply:
Will change to return the Id only